corehq/ex-submodules/phonelog/utils.py

```diff
@@ -191,11 +191,14 @@ def __init__(self, domain, xform):
         self.xform = xform
 
     def send_data(self, url):
-        send_device_log_to_sumologic.delay(url, self.log_subreport(), self._get_header('log'))
-        send_device_log_to_sumologic.delay(url, self.user_error_subreport(), self._get_header('user_error'))
-        send_device_log_to_sumologic.delay(url, self.force_close_subreport(), self._get_header('force_close'))
-
-    def _get_header(self, fmt):
+        send_device_log_to_sumologic.delay(url, self.log_subreport(should_encode=False),
+                                           self._get_header('log', False))
+        send_device_log_to_sumologic.delay(url, self.user_error_subreport(should_encode=False),
+                                           self._get_header('user_error', False))
+        send_device_log_to_sumologic.delay(url, self.force_close_subreport(should_encode=False),
```

Comment:
I'm regretting my `should_encode` suggestion because it looks like this method is always called with should_encode=False in production code, and therefore we should just remove this parameter. Sorry for the confusion (October me didn't know what he was talking about)

Reply:
They've been removed. There were tests that referenced these method before (when it returned an encoded string) but somehow still pass despite not making any changes to them.